src/modules/mavlink/streams/HOME_POSITION.hpp

```diff
@@ -75,7 +75,7 @@ class MavlinkStreamHomePosition : public MavlinkStream
 				msg.y = home.y;
 				msg.z = home.z;
 
-				matrix::Quatf q(matrix::Eulerf(0.f, 0.f, home.yaw));
+				matrix::Quatf q(matrix::Eulerf(home.roll, home.pitch, home.yaw));
```

Comment:
Can this be streamed before roll or pitch are set, or if they are NaN? If so, will this break?

Reply:
ie. if we call `fillLocalHomePos(home, home_x, home_y, home_z, NAN, NAN, NAN);` and this is streamed, what happens?

Reply:
It will send out NAN for all quaternion fields.